react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -504,6 +513,11 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                             oldText,
                             start,
                             start + before));
+
+            // Add the outer tags when the field was started empty, and only the first time the user types in it.
+            if (mPreviousText.length() == 0 && currentEventCount == 1 && !TextUtils.isEmpty(mEditText.getTagName())) {
+                mEditText.fromHtml('<' + mEditText.getTagName() + '>' + newText + "</" + mEditText.getTagName() + '>', false);
```

Comment:
Is there maybe some Util that we can use to generate the `html` from tag name and text in order to have one place for adding the `<,>` when there is a need for them?

Reply:
Not that i'm aware of, but in this case it's pretty simple the input string, that doesn't worth  introduce a new function for it. We can use StringBuilder, but it's a very simple string.